spotifytracks.py

```diff
@@ -77,6 +101,54 @@ def get_playlist_tracks(self, playlist_id: str, limit: int = None):
 
             offset += 50
 
+    def get_album_tracks(self, album_id:str, limit:int = None):
+        ''' Get a list of tracks of a playlist.
+
+        Parameters:
+            - playlist_id: the id of the playlist
+        '''
+        album_query = f"https://api.spotify.com/v1/albums/{album_id}"
+        album_data_response = requests.get(
+            url=album_query,
+            headers={
+                "Content-Type": "application/json",
+                "Authorization": f'Bearer {token}'
+            }
+        )
+        album_data_response = album_data_response.json()
+        offset = 0
+        if limit is None:
+            limit = 10000
+        fetched = 0
+        while offset < limit:
+            query = f"https://api.spotify.com/v1/albums/{album_id}/tracks?offset={offset}&limit=50"
+            response = requests.get(
```

Comment:
This request to fetch the album tracks is redundant since album_data_response already contains all the track information along with the album information.

Please let me know if there's a reason you did it this way, or if I'm missing something.

Reply:
You're right it does contain the tracks but only 0-50, I hadn't realized at the time since the spotify docs labels it "Get album data" and "Get album tracks" so I had assumed it was only album data. I've fixed it now.